content/challenges/easy/9 - Factorial of a Positive Integer/func.js

```diff
@@ -12,5 +12,16 @@
  * @returns {number}
  */
 module.exports = (num) => {
-   // Your code goes here
+   if (num < 0) {
+      throw new Error("No Negatives");
```

Comment:
Writing error statements is to know your audience. Your audience is other developers or folks in IT. I think something like `Cannot pass in a negative number` is better. This is the type of things that shows up in logs that folks trying to debug are looking at. Succinct but meaningful messages are SUPER important when it comes to error handling! It could mean the difference between finding the solution to a production but in minutes or HOURS!